### PR TITLE
Completed css-flexbox

### PR DESCRIPTION
**Pull request by Eugenepark215**

### CODE REVIEW

File: css-flexbox/justify-content.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+#card-deck {
+  display: flex;
+  justify-content: center;
+  flex-basis: calc(100%/4);
```
Comment (thebearingedge) [inline css-flexbox/justify-content.html:14]: `flex-basis` only works on "flex items" (the children of elements with `display: flex`), so you should move `flex-basis` to the `.card-wrapper`

File: css-flexbox/display-flex.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+#card-deck {
+  display: flex;
+  padding: 20px;
+}
+.pokemon-card {
+  padding: 20px;
+  margin: 20px;
+}
```
Comment (thebearingedge) [inline css-flexbox/display-flex.html:18]: You can remove this, we don't need to change the styles for the `.pokemon-card`

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please add screenshots and also see my notes.

**Comment by Eugenepark215** [comment]
Screen Shot 2022-08-25 at 8 41 54 AM (2)
Screen Shot 2022-08-25 at 8 41 54 AM
Screen Shot 2022-08-25 at 8 42 02 AM
Screen Shot 2022-08-25 at 8 42 02 AM (2)

**Pull request merged**